lldb/source/Plugins/Process/minidump/ProcessMinidump.cpp

```diff
@@ -209,7 +208,20 @@ Status ProcessMinidump::DoLoadCore() {
   GetTarget().SetArchitecture(arch, true /*set_platform*/);
 
   m_thread_list = m_minidump_parser->GetThreads();
-  m_active_exception = m_minidump_parser->GetExceptionStream();
+  std::vector<minidump::ExceptionStream> exception_streams =
```

Comment:
Do not return by a vector of copies, instead return a value of const references/pointers.